src_c/alphablit.c

```diff
@@ -265,7 +265,7 @@ SoftBlitPyGame(SDL_Surface *src, SDL_Rect *srcrect, SDL_Surface *dst,
                     break;
                 }
                 case PYGAME_BLEND_ADD: {
-#if !defined(__EMSCRIPTEN__)
+#if !defined(__EMSCRIPTEN__) && !defined(_M_ARM64)
 #if SDL_BYTEORDER == SDL_LIL_ENDIAN
```

Comment:
What happens if you don't add this?

Reply:
If you don't, he will get a LINK error and will not be able to parse the symbols.
Because these symbols cannot be used on Windows arm.

Reply:
I'm curious what symbols?

Reply:
Some symbols about RGB functions, such as blit_blend_rgb_sub_sse2.

After my analysis, the reasons for incompatibility are as follows: First, MSVC ARM64 does not support SSE, which is a fact. In this file, there is already a way to be compatible with ARM64, even with sse2neon, in the GitHub repository of this header file, I found an (https://github.com/DLTcollab/sse2neon/issues/384), which means that this header file does not support MSVC.